pom.xml

```diff
@@ -43,6 +45,110 @@
     <guava.version>19.0</guava.version>
   </properties>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.5.1</version>
+          <configuration>
+            <source>1.6</source>
+            <target>1.6</target>
+          </configuration>
+          <executions>
+            <execution>
+              <id>default-testCompile</id>
+              <phase>test-compile</phase>
+              <goals>
+                <goal>testCompile</goal>
+              </goals>
+              <configuration>
+                <skip>true</skip>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-jar-plugin</artifactId>
+          <version>2.3.1</version>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-source-plugin</artifactId>
+          <version>2.1.2</version>
+          <executions>
+            <execution>
+              <id>attach-sources</id>
+              <goals>
+                <goal>jar-no-fork</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <version>2.8.1</version>
+          <executions>
+            <execution>
+              <id>attach-javadocs</id>
+              <goals>
+                <goal>jar</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>2.17</version>
+          <dependencies>
+            <dependency>
+              <groupId>com.puppycrawl.tools</groupId>
+              <artifactId>checkstyle</artifactId>
+              <version>7.6</version>
+            </dependency>
+          </dependencies>
+          <executions>
+            <execution>
+              <id>validate</id>
+              <phase>validate</phase>
+              <configuration>
+                <configLocation>google_checks.xml</configLocation>
+                <encoding>UTF-8</encoding>
+                <consoleOutput>true</consoleOutput>
+                <failsOnError>true</failsOnError>
```

Comment:
I cannot add that for the moment. Added but commented and we will enable when we fix the violations (in a separate PR):

[WARNING] src/main/java/com/google/instrumentation/stats/DistributionAggregation.java:[24] (javadoc) JavadocParagraph: <p> tag should be placed immediately before the first word, with no space after.
[WARNING] src/main/java/com/google/instrumentation/stats/IntervalAggregationDescriptor.java:[29] (javadoc) JavadocParagraph: <p> tag should be placed immediately before the first word, with no space after.
[WARNING] src/main/java/com/google/instrumentation/stats/IntervalAggregationDescriptor.java:[32] (javadoc) JavadocParagraph: <p> tag should be placed immediately before the first word, with no space after.
[WARNING] src/main/java/com/google/instrumentation/stats/IntervalAggregationDescriptor.java:[59] (javadoc) JavadocParagraph: <p> tag should be placed immediately before the first word, with no space after.
[WARNING] src/main/java/com/google/instrumentation/stats/StatsContextFactory.java:[17] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.IOException' import. Should be before 'java.io.InputStream'.
[WARNING] src/main/java/com/google/instrumentation/stats/ViewDescriptor.java:[59] (indentation) CommentsIndentation: Block comment has incorrect indentation level 1, expected is 2, indentation should be the same level as line 62.